docs/source/cli_help/status.rst

```diff
@@ -80,28 +81,7 @@ With ``--color=always``, git machete always emits colors and with ``--color=auto
 
 --no-detect-squash-merges         Only consider *strict* (fast-forward or 2-parent) merges, rather than rebase/squash merges, when detecting if a branch is merged into its upstream (parent).
 
- .. note::
-
-    To make it easier to select branch name from the ``status`` output on certain terminals
-    (e.g. `Alacritty <https://github.com/alacritty/alacritty>`_), you can add an extra
-    space between ``└─`` and ``branch name`` by setting ``git config machete.status.extraSpaceBeforeBranchName true``.
```

Comment:
Nit: maybe I'd keep the elbow un-quoted:

![image](https://user-images.githubusercontent.com/3383210/189219875-d8029902-a410-43a8-a733-1f6900ffd98f.png)

this underline looks weird in terminal